patches/ui-views-layout-layout_provider.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ui/views/layout/layout_provider.h b/ui/views/layout/layout_provider.h
+index d40bdd14cf7e719082a33189730153f74888644d..65f60d2f3650c534ef9235476dc0825497bcfbef 100644
+--- a/ui/views/layout/layout_provider.h
++++ b/ui/views/layout/layout_provider.h
+@@ -228,7 +228,7 @@ class VIEWS_EXPORT LayoutProvider {
+   // Returns the corner radius related to a specific context token.
+   // TODO(crbug.com/40255130): Replace GetCornerRadiusMetric(Emphasis...) with
+   // context tokens.
+-  int GetCornerRadiusMetric(ShapeContextTokens token,
++  virtual int GetCornerRadiusMetric(ShapeContextTokens token,
```

Comment:
Need to override `GetCornerRAdiusMetric` from our subclass but upstream has another same name method with different params. and it's already `virtual`. So `#define GetCornerRadiusMetric virtual GetCornerRAdiusMetric` doesn't work.

Reply:
why do we need to override this one? Doesn't it defer to the other one for basically everything when `IsChromeRefresh2023` is not enabled?

Reply:
https://google.github.io/styleguide/cppguide.html#Default_Arguments `Default arguments are banned on virtual functions, where they don't work properly`

we could just patch the cc file and do the same thing without the above issue so you might as well just patch directly. This is going to change soon anyway with `IsChromeRefresh2023`

Reply:
Addressed - c82707a4c30d30d8b23c3c8420ce471e86e89d97